cpp/tests/io/parquet_test.cpp

```diff
@@ -357,6 +357,11 @@ struct ParquetWriterSchemaTest : public ParquetWriterTest {
   auto type() { return cudf::data_type{cudf::type_to_id<T>()}; }
 };
 
+template <typename T>
+struct ParquetReaderSourceTest : public ParquetReaderTest {
+  auto type() { return cudf::data_type{cudf::type_to_id<T>()}; }
```

Comment:
I don't think this is needed, since the type does not refer to the data type in the test.
```suggestion
```